### PR TITLE
UCT/IB: Fix force global AV

### LINKED ISSUE #3020: [mtt] Assertion `gid->global.interface_id != 0' failed
Configuration
```
OMPI: 4.0.0rc5
MOFED: MLNX_OFED_LINUX-4.4-2.0.7.0
Module: hpcx-gcc (2018-11-03)
Test module: mtt-tests/hpcx-gcc
Nodes: polar-sriov x4 (ppn=6(x4), nodelist=hpc-polar-01-vm[001-002],hpc-polar-02-vm[001-002])
```

MTT log: http://hpcweb.lab.mtl.com/hpc/mtr_scrap/users/mtt/scratch/ucx_ompi/20181104_102627_14859_11241_hpc-polar-01-vm001/html/test_stdout_JGyMwA.txt

Cmd:
`mpirun -np 2 --display-map -mca btl self --tag-output --timestamp-output -mca pml ucx -mca coll '^hcoll' --bind-to core -x UCX_NET_DEVICES=mlx5_0:1 -mca osc ucx -x UCX_IB_ADDR_TYPE=ib_global -x UCX_IB_REG_METHODS=rcache,direct -x UCX_TLS=rc_x,self,sm -x UCX_RC_MLX5_TM_ENABLE=n -mca pmix_base_async_modex 0 -mca mpi_add_procs_cutoff 100000 -mca pmix_base_collect_data 1 --map-by node /hpc/local/benchmarks/hpcx_install_2018-11-03/mtt-tests-gcc/installs/qVUg/tests/osu_micro_benchmark/osu-micro-benchmarks-5.3.2/mpi/one-sided/osu_get_acc_latency`

Output:
```sh
Sun Nov  4 10:28:56 2018[1,1]<stdout>:[1541320136.258417] [hpc-polar-01-vm002:24375:0]            cpu.c:53   UCX  WARN  CPU does not support invariant TSC, time may be unstable
Sun Nov  4 10:28:56 2018[1,1]<stderr>:[hpc-polar-01-vm002:24375:0:24375]    ib_iface.h:496  UCX Assertion `gid->global.interface_id != 0' failed
Sun Nov  4 10:28:56 2018[1,1]<stderr>:
Sun Nov  4 10:28:56 2018[1,1]<stderr>:/hpc/local/benchmarks/hpcx_install_2018-11-03/src/hpcx-gcc-redhat7.4/ucx-master/src/uct/ib/base/ib_iface.h: [ uct_ib_iface_fill_ah_attr_from_gid_lid() ]
Sun Nov  4 10:28:56 2018[1,1]<stderr>:      ...Sun Nov  4 10:28:56 2018[1,1]<stderr>:      493     if (iface->is_global_addr ||
Sun Nov  4 10:28:56 2018[1,1]<stderr>:      494         (iface->gid.global.subnet_prefix != gid->global.subnet_prefix)) {
Sun Nov  4 10:28:56 2018[1,1]<stderr>:      495         ucs_assert_always(gid->global.interface_id != 0);
Sun Nov  4 10:28:56 2018[1,1]<stderr>:==>   496         ah_attr->is_global      = 1;
Sun Nov  4 10:28:56 2018[1,1]<stderr>:      497         ah_attr->grh.dgid       = *gid;
Sun Nov  4 10:28:56 2018[1,1]<stderr>:      498         ah_attr->grh.sgid_index = iface->config.gid_index;
Sun Nov  4 10:28:56 2018[1,1]<stderr>:      499         ah_attr->grh.hop_limit  = iface->config.hop_limit;
Sun Nov  4 10:28:56 2018[1,1]<stderr>:Sun Nov  4 10:28:56 2018[1,1]<stderr>:==== backtrace ====
Sun Nov  4 10:28:56 2018[1,1]<stderr>: 0 0x000000000002d7b2 uct_ib_iface_fill_ah_attr_from_gid_lid()  /hpc/local/benchmarks/hpcx_install_2018-11-03/src/hpcx-gcc-redhat7.4/ucx-master/src/uct/ib/base/ib_iface.h:496
Sun Nov  4 10:28:56 2018[1,1]<stderr>: 1 0x000000000005c02d uct_ud_mlx5_iface_t_init()  /hpc/local/benchmarks/hpcx_install_2018-11-03/src/hpcx-gcc-redhat7.4/ucx-master/src/uct/ib/ud/accel/ud_mlx5.c:721
Sun Nov  4 10:28:56 2018[1,1]<stderr>: 2 0x000000000001f3f5 uct_iface_open()  /hpc/local/benchmarks/hpcx_install_2018-11-03/src/hpcx-gcc-redhat7.4/ucx-master/src/uct/base/uct_md.c:352
Sun Nov  4 10:28:56 2018[1,1]<stderr>: 3 0x000000000001689e ucp_worker_iface_init()  /hpc/local/benchmarks/hpcx_install_2018-11-03/src/hpcx-gcc-redhat7.4/ucx-master/src/ucp/core/ucp_worker.c:844
Sun Nov  4 10:28:56 2018[1,1]<stderr>: 4 0x0000000000017659 ucp_worker_add_resource_ifaces()  /hpc/local/benchmarks/hpcx_install_2018-11-03/src/hpcx-gcc-redhat7.4/ucx-master/src/ucp/core/ucp_worker.c:774
Sun Nov  4 10:28:56 2018[1,1]<stderr>: 5 0x00000000000059c4 mca_pml_ucx_init()  /hpc/local/benchmarks/hpcx_install_2018-11-03/src/hpcx-gcc-redhat7.4/ompi/ompi/mca/pml/ucx/pml_ucx.c:209
Sun Nov  4 10:28:56 2018[1,1]<stderr>: 6 0x0000000000007719 mca_pml_ucx_component_init()  /hpc/local/benchmarks/hpcx_install_2018-11-03/src/hpcx-gcc-redhat7.4/ompi/ompi/mca/pml/ucx/pml_ucx_component.c:97
Sun Nov  4 10:28:56 2018[1,1]<stderr>: 7 0x00000000000a3a26 mca_pml_base_select()  /hpc/local/benchmarks/hpcx_install_2018-11-03/src/hpcx-gcc-redhat7.4/ompi/ompi/mca/pml/base/pml_base_select.c:126
Sun Nov  4 10:28:56 2018[1,1]<stderr>: 8 0x000000000004d350 ompi_mpi_init()  /hpc/local/benchmarks/hpcx_install_2018-11-03/src/hpcx-gcc-redhat7.4/ompi/ompi/runtime/ompi_mpi_init.c:646
Sun Nov  4 10:28:56 2018[1,1]<stderr>: 9 0x000000000006ac0b PMPI_Init()  /hpc/local/benchmarks/hpcx_install_2018-11-03/src/hpcx-gcc-redhat7.4/ompi/ompi/mpi/c/profile/pinit.c:67
Sun Nov  4 10:28:56 2018[1,1]<stderr>:10 0x00000000004014eb main()  /hpc/local/benchmarks/hpcx_install_2018-11-03/mtt-tests-gcc/installs/qVUg/tests/osu_micro_benchmark/osu-micro-benchmarks-5.3.2/mpi/one-sided/osu_get_acc_latency.c:122
Sun Nov  4 10:28:56 2018[1,1]<stderr>:11 0x0000000000021c05 __libc_start_main()  ???:0
Sun Nov  4 10:28:56 2018[1,1]<stderr>:12 0x0000000000401795 _start()  ???:0
Sun Nov  4 10:28:56 2018[1,1]<stderr>:===================
Sun Nov  4 10:28:56 2018[1,1]<stderr>:[hpc-polar-01-vm002:24375] *** Process received signal ***
Sun Nov  4 10:28:56 2018[1,1]<stderr>:[hpc-polar-01-vm002:24375] Signal: Aborted (6)
Sun Nov  4 10:28:56 2018[1,1]<stderr>:[hpc-polar-01-vm002:24375] Signal code:  (-6)
Sun Nov  4 10:28:56 2018[1,1]<stderr>:[hpc-polar-01-vm002:24375] [ 0] /usr/lib64/libpthread.so.0(+0xf5e0)[0x7f04431015e0]
Sun Nov  4 10:28:56 2018[1,1]<stderr>:[hpc-polar-01-vm002:24375] [ 1] /usr/lib64/libc.so.6(gsignal+0x37)[0x7f0442d641f7]
Sun Nov  4 10:28:56 2018[1,1]<stderr>:[hpc-polar-01-vm002:24375] [ 2] /usr/lib64/libc.so.6(abort+0x148)[0x7f0442d658e8]
Sun Nov  4 10:28:56 2018[1,1]<stderr>:[hpc-polar-01-vm002:24375] [ 3] /hpc/local/benchmarks/hpcx_install_2018-11-03/hpcx-gcc-redhat7.4/ucx/lib/libucs.so.0(+0x46e51)[0x7f0438118e51]
Sun Nov  4 10:28:56 2018[1,1]<stderr>:[hpc-polar-01-vm002:24375] [ 4] /hpc/local/benchmarks/hpcx_install_2018-11-03/hpcx-gcc-redhat7.4/ucx/lib/libuct.so.0(+0x2d7b2)[0x7f0438cdd7b2]
Sun Nov  4 10:28:56 2018[1,1]<stderr>:[hpc-polar-01-vm002:24375] [ 5] /hpc/local/benchmarks/hpcx_install_2018-11-03/hpcx-gcc-redhat7.4/ucx/lib/libuct.so.0(+0x5c02d)[0x7f0438d0c02d]
Sun Nov  4 10:28:56 2018[1,1]<stderr>:[hpc-polar-01-vm002:24375] [ 6] /hpc/local/benchmarks/hpcx_install_2018-11-03/hpcx-gcc-redhat7.4/ucx/lib/libuct.so.0(uct_iface_open+0xb5)[0x7f0438ccf3f5]
Sun Nov  4 10:28:56 2018[1,1]<stderr>:[hpc-polar-01-vm002:24375] [ 7] /hpc/local/benchmarks/hpcx_install_2018-11-03/hpcx-gcc-redhat7.4/ucx/lib/libucp.so.0(ucp_worker_iface_init+0x20e)[0x7f0438f4d89e]
Sun Nov  4 10:28:56 2018[1,1]<stderr>:[hpc-polar-01-vm002:24375] [ 8] /hpc/local/benchmarks/hpcx_install_2018-11-03/hpcx-gcc-redhat7.4/ucx/lib/libucp.so.0(ucp_worker_create+0x3e9)[0x7f0438f4e659]
Sun Nov  4 10:28:56 2018[1,1]<stderr>:[hpc-polar-01-vm002:24375] [ 9] /hpc/local/benchmarks/hpcx_install_2018-11-03/hpcx-gcc-redhat7.4/ompi/lib/openmpi/mca_pml_ucx.so(mca_pml_ucx_init+0x84)[0x7f04393879c4]
Sun Nov  4 10:28:56 2018[1,1]<stderr>:[hpc-polar-01-vm002:24375] [10] /hpc/local/benchmarks/hpcx_install_2018-11-03/hpcx-gcc-redhat7.4/ompi/lib/openmpi/mca_pml_ucx.so(+0x7719)[0x7f0439389719]
Sun Nov  4 10:28:56 2018[1,1]<stderr>:[hpc-polar-01-vm002:24375] [11] /hpc/local/benchmarks/hpcx_install_2018-11-03/hpcx-gcc-redhat7.4/ompi/lib/libmpi.so.40(mca_pml_base_select+0x1c6)[0x7f04433b1a26]
Sun Nov  4 10:28:56 2018[1,1]<stderr>:[hpc-polar-01-vm002:24375] [12] /hpc/local/benchmarks/hpcx_install_2018-11-03/hpcx-gcc-redhat7.4/ompi/lib/libmpi.so.40(ompi_mpi_init+0x740)[0x7f044335b350]
Sun Nov  4 10:28:56 2018[1,1]<stderr>:[hpc-polar-01-vm002:24375] [13] /hpc/local/benchmarks/hpcx_install_2018-11-03/hpcx-gcc-redhat7.4/ompi/lib/libmpi.so.40(MPI_Init+0x9b)[0x7f0443378c0b]
Sun Nov  4 10:28:56 2018[1,1]<stderr>:[hpc-polar-01-vm002:24375] [14] /hpc/local/benchmarks/hpcx_install_2018-11-03/mtt-tests-gcc/installs/qVUg/tests/oSun Nov  4 10:28:56 2018[1,1]<stderr>:su_micro_benchmark/osu-micro-benchmarks-5.3.2/mpi/one-sided/osu_get_acc_latency[0x4014eb]
Sun Nov  4 10:28:56 2018[1,1]<stderr>:[hpc-polar-01-vm002:24375] [15] /usr/lib64/libc.so.6(__libc_start_main+0xf5)[0x7f0442d50c05]
Sun Nov  4 10:28:56 2018[1,1]<stderr>:[hpc-polar-01-vm002:24375] [16] /hpc/local/benchmarks/hpcx_install_2018-11-03/mtt-tests-gcc/installs/qVUg/tests/osu_micro_benchmark/osu-micro-benchmarks-5.3.2/mpi/one-sided/osu_get_acc_latency[0x401795]
Sun Nov  4 10:28:56 2018[1,1]<stderr>:[hpc-polar-01-vm002:24375] *** End of error message ***--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------Sun Nov  4 10:28:58 2018[1,0]<stdout>:[1541320137.240574] [hpc-polar-01-vm001:3738 :0]            cpu.c:53   UCX  WARN  CPU does not support invariant TSC, time may be unstable
Sun Nov  4 10:28:58 2018[1,0]<stderr>:[hpc-polar-01-vm001:3738 :0:3738]    ib_iface.h:496  UCX Assertion `gid->global.interface_id != 0' failed
Sun Nov  4 10:28:58 2018[1,0]<stderr>:
Sun Nov  4 10:28:58 2018[1,0]<stderr>:/hpc/local/benchmarks/hpcx_install_2018-11-03/src/hpcx-gcc-redhat7.4/ucx-master/src/uct/ib/base/ib_iface.h: [ uct_ib_iface_fill_ah_attr_from_gid_lid() ]
Sun Nov  4 10:28:58 2018[1,0]<stderr>:      ...
Sun Nov  4 10:28:58 2018[1,0]<stderr>:      493     if (iface->is_global_addr ||
Sun Nov  4 10:28:58 2018[1,0]<stderr>:      494         (iface->gid.global.subnet_prefix != gid->global.subnet_prefix)) {
Sun Nov  4 10:28:58 2018[1,0]<stderr>:      495         ucs_assert_always(gid->global.interface_id != 0);
Sun Nov  4 10:28:58 2018[1,0]<stderr>:==>   496         ah_attr->is_global      = 1;
Sun Nov  4 10:28:58 2018[1,0]<stderr>:      497         ah_attr->grh.dgid       = *gid;
Sun Nov  4 10:28:58 2018[1,0]<stderr>:      498         ah_attr->grh.sgid_index = iface->config.gid_index;
Sun Nov  4 10:28:58 2018[1,0]<stderr>:      499         ah_attr->grh.hop_limit  = iface->config.hop_limit;
Sun Nov  4 10:28:58 2018[1,0]<stderr>:
Sun Nov  4 10:28:58 2018[1,0]<stderr>:==== backtrace ====
Sun Nov  4 10:28:58 2018[1,0]<stderr>: 0 0x000000000002d7b2 uct_ib_iface_fill_ah_attr_from_gid_lid()  /hpc/local/benchmarks/hpcx_install_2018-11-03/src/hpcx-gcc-redhat7.4/ucx-master/src/uct/ib/base/ib_iface.h:496
Sun Nov  4 10:28:58 2018[1,0]<stderr>: 1 0x000000000005c02d uct_ud_mlx5_iface_t_init()  /hpc/local/benchmarks/hpcx_install_2018-11-03/src/hpcx-gcc-redhat7.4/ucx-master/src/uct/ib/ud/accel/ud_mlx5.c:721
Sun Nov  4 10:28:58 2018[1,0]<stderr>: 2 0x000000000001f3f5 uct_iface_open()  /hpc/local/benchmarks/hpcx_install_2018-11-03/src/hpcx-gcc-redhat7.4/ucx-master/src/uct/base/uct_md.c:352
Sun Nov  4 10:28:58 2018[1,0]<stderr>: 3 0x000000000001689e ucp_worker_iface_init()  /hpc/local/benchmarks/hpcx_install_2018-11-03/src/hpcx-gcc-redhat7.4/ucx-master/src/ucp/core/ucp_worker.c:844
Sun Nov  4 10:28:58 2018[1,0]<stderr>: 4 0x0000000000017659 ucp_worker_add_resource_ifaces()  /hpc/local/benchmarks/hpcx_install_2018-11-03/src/hpcx-gcc-redhat7.4/ucx-master/src/ucp/core/ucp_worker.c:774
Sun Nov  4 10:28:58 2018[1,0]<stderr>: 5 0x00000000000059c4 mca_pml_ucx_init()  /hpc/local/benchmarks/hpcx_install_2018-11-03/src/hpcx-gcc-redhat7.4/ompi/ompi/mca/pml/ucx/pml_ucx.c:209
Sun Nov  4 10:28:58 2018[1,0]<stderr>: 6 0x0000000000007719 mca_pml_ucx_component_init()  /hpc/local/benchmarks/hpcx_install_2018-11-03/src/hpcx-gcc-redhat7.4/ompi/ompi/mca/pml/ucx/pml_ucx_component.c:97
Sun Nov  4 10:28:58 2018[1,0]<stderr>: 7 0x00000000000a3a26 mca_pml_base_select()  /hpc/local/benchmarks/hpcx_install_2018-11-03/src/hpcx-gcc-redhat7.4/ompi/ompi/mca/pml/base/pml_base_select.c:126
Sun Nov  4 10:28:58 2018[1,0]<stderr>: 8 0x000000000004d350 ompi_mpi_init()  /hpc/local/benchmarks/hpcx_install_2018-11-03/src/hpcx-gcc-redhat7.4/ompi/ompi/runtime/ompi_mpi_init.c:646
Sun Nov  4 10:28:58 2018[1,0]<stderr>: 9 0x000000000006ac0b PMPI_Init()  /hpc/local/benchmarks/hpcx_install_2018-11-03/src/hpcx-gcc-redhat7.4/ompi/ompi/mpi/c/profile/pinit.c:67
Sun Nov  4 10:28:58 2018[1,0]<stderr>:10 0x00000000004014eb main()  /hpc/local/benchmarks/hpcx_install_2018-11-03/mtt-tests-gcc/installs/qVUg/tests/osu_micro_benchmark/osu-micro-benchmarks-5.3.2/mpi/one-sided/osu_get_acc_latency.c:122
Sun Nov  4 10:28:58 2018[1,0]<stderr>:11 0x0000000000021c05 __libc_start_main()  ???:0
Sun Nov  4 10:28:58 2018[1,0]<stderr>:12 0x0000000000401795 _start()  ???:0
Sun Nov  4 10:28:58 2018[1,0]<stderr>:===================
Sun Nov  4 10:28:58 2018[1,0]<stderr>:[hpc-polar-01-vm001:03738] *** Process received signal ***
Sun Nov  4 10:28:58 2018[1,0]<stderr>:[hpc-polar-01-vm001:03738] Signal: Aborted (6)
Sun Nov  4 10:28:58 2018[1,0]<stderr>:[hpc-polar-01-vm001:03738] Signal code:  (-6)
Sun Nov  4 10:28:58 2018[1,0]<stderr>:[hpc-polar-01-vm001:03738] [ 0] /usr/lib64/libpthread.so.0(+0xf5e0)[0x7fab36fbb5e0]
Sun Nov  4 10:28:58 2018[1,0]<stderr>:[hpc-polar-01-vm001:03738] [ 1] /usr/lib64/libc.so.6(gsignal+0x37)[0x7fab36c1e1f7]
Sun Nov  4 10:28:58 2018[1,0]<stderr>:[hpc-polar-01-vm001:03738] [ 2] /usr/lib64/libc.so.6(abort+0x148)[0x7fab36c1f8e8]
Sun Nov  4 10:28:58 2018[1,0]<stderr>:[hpc-polar-01-vm001:03738] [ 3] /hpc/local/benchmarks/hpcx_install_2018-11-03/hpcx-gcc-redhat7.4/ucx/lib/libucs.so.0(+0x46e51)[0x7fab27cfde51]
Sun Nov  4 10:28:58 2018[1,0]<stderr>:[hpc-polar-01-vm001:03738] [ 4] /hpc/local/benchmarks/hpcx_install_2018-11-03/hpcx-gcc-redhat7.4/ucx/lib/libuct.so.0(+0x2d7b2)[0x7fab2cb977b2]
Sun Nov  4 10:28:58 2018[1,0]<stderr>:[hpc-polar-01-vm001:03738] [ 5] /hpc/local/benchmarks/hpcx_install_2018-11-03/hpcx-gcc-redhat7.4/ucx/lib/libuct.so.0(+0x5c02d)[0x7fab2cbc602d]
Sun Nov  4 10:28:58 2018[1,0]<stderr>:[hpc-polar-01-vm001:03738] [ 6] /hpc/local/benchmarks/hpcx_install_2018-11-03/hpcx-gcc-redhat7.4/ucx/lib/libuct.so.0(uct_iface_open+0xb5)[0x7fab2cb893f5]
Sun Nov  4 10:28:58 2018[1,0]<stderr>:[hpc-polar-01-vm001:03738] [ 7] /hpc/local/benchmarks/hpcx_install_2018-11-03/hpcx-gcc-redhat7.4/ucx/lib/libucp.so.0(ucp_worker_iface_init+0x20e)[0x7fab2ce0789e]
Sun Nov  4 10:28:58 2018[1,0]<stderr>:[hpc-polar-01-vm001:03738] [ 8] /hpc/local/benchmarks/hpcx_install_2018-11-03/hpcx-gcc-redhat7.4/ucx/lib/libucp.so.0(ucp_worker_create+0x3e9)[0x7fab2ce08659]
Sun Nov  4 10:28:58 2018[1,0]<stderr>:[hpc-polar-01-vm001:03738] [ 9] /hpc/local/benchmarks/hpcx_install_2018-11-03/hpcx-gcc-redhat7.4/ompi/lib/openmpi/mca_pml_ucx.so(mca_pml_ucx_init+0x84)[0x7fab2d2419c4]
Sun Nov  4 10:28:58 2018[1,0]<stderr>:[hpc-polar-01-vm001:03738] [10] /hSun Nov  4 10:28:58 2018[1,0]<stderr>:pc/local/benchmarks/hpcx_install_2018-11-03/hpcx-gcc-redhat7.4/ompi/lib/openmpi/mca_pml_ucx.so(+0x7719)[0x7fab2d243719]
Sun Nov  4 10:28:58 2018[1,0]<stderr>:[hpc-polar-01-vm001:03738] [11] /hpc/local/benchmarks/hpcx_install_2018-11-03/hpcx-gcc-redhat7.4/ompi/lib/libmpi.so.40(mca_pml_base_select+0x1c6)[0x7fab3726ba26]
Sun Nov  4 10:28:58 2018[1,0]<stderr>:[hpc-polar-01-vm001:03738] [12] /hpc/local/benchmarks/hpcx_install_2018-11-03/hpcx-gcc-redhat7.4/ompi/lib/libmpi.so.40(ompi_mpi_init+0x740)[0x7fab37215350]
Sun Nov  4 10:28:58 2018[1,0]<stderr>:[hpc-polar-01-vm001:03738] [13] /hpc/local/benchmarks/hpcx_install_2018-11-03/hpcx-gcc-redhat7.4/ompi/lib/libmpi.so.40(MPI_Init+0x9b)[0x7fab37232c0b]
Sun Nov  4 10:28:58 2018[1,0]<stderr>:[hpc-polar-01-vm001:03738] [14] /hpc/local/benchmarks/hpcx_install_2018-11-03/mtt-tests-gcc/installs/qVUg/tests/osu_micro_benchmark/osu-micro-benchmarks-5.3.2/mpi/one-sided/osu_get_acc_latency[0x4014eb]
Sun Nov  4 10:28:58 2018[1,0]<stderr>:[hpc-polar-01-vm001:03738] [15] /usr/lib64/libc.so.6(__libc_start_main+0xf5)[0x7fab36c0ac05]
Sun Nov  4 10:28:58 2018[1,0]<stderr>:[hpc-polar-01-vm001:03738] [16] /hpc/local/benchmarks/hpcx_install_2018-11-03/mtt-tests-gcc/installs/qVUg/tests/osu_micro_benchmark/osu-micro-benchmarks-5.3.2/mpi/one-sided/osu_get_acc_latency[0x401795]
Sun Nov  4 10:28:58 2018[1,0]<stderr>:[hpc-polar-01-vm001:03738] *** End of error message ***--------------------------------------------------------------------------
mpirun noticed that process rank 1 with PID 24375 on node hpc-polar-01-vm002 exited on signal 6 (Aborted).
```

### PR DESCRIPTION
**Pull request by Artemy-Mellanox**
Fixes #3020

### CODE REVIEW

File: src/uct/ib/base/ib_iface.c [hunk 1]
```diff
@@ -219,6 +219,11 @@ size_t uct_ib_address_size(uct_ib_iface_t *iface)
     if (IBV_PORT_IS_LINK_LAYER_ETHERNET(uct_ib_iface_port_attr(iface))) {
         return sizeof(uct_ib_address_t) +
                sizeof(union ibv_gid);  /* raw gid */
+    } else if (iface->is_global_addr) {
```
Comment (yosefe) [inline src/uct/ib/base/ib_iface.c]: maybe add && !iface->is_global_addr to next condition, to unite this with last case (else)?

File: src/uct/ib/base/ib_iface.c [hunk 2]
```diff
@@ -248,6 +253,17 @@ void uct_ib_address_pack(uct_ib_iface_t *iface,
                          UCT_IB_ADDRESS_FLAG_GID;
         /* uint8_t raw[16]; */
         memcpy(ptr, gid->raw, sizeof(gid->raw) * sizeof(uint8_t));
+    } else if (iface->is_global_addr) {
```
Comment (yosefe) [inline src/uct/ib/base/ib_iface.c]: let's unite with already existing "global" case, in "else"

**Comment by Artemy-Mellanox** [comment]
Yes, test passed on this setup.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8202/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8203/ for details (Mellanox internal link).

**Pull request merged**